src/core/runtime/ipld-browser.js

```diff
@@ -0,0 +1,8 @@
+'use strict'
+const mergeOptions = require('merge-options')
+
+module.exports = (blockService, options = {}) => {
+  return mergeOptions({
+    blockService: blockService
+  }, options)
+}
```

Comment:
Until we have multiple browser builds we need to include bitcoin, ethereum, git and zcash.